components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.html

```diff
@@ -14,27 +14,87 @@
     <div *ngIf="!loading">
       <chef-modal label="member-expression-label" class="member-expression-modal" [visible]="modalVisible" (close)="closeModal()">
         <h2>Add Member Expression</h2>
-        <div id="left-align">
+        <div id="modal-content-container">
           <form [formGroup]="expressionForm">
+
+              <chef-form-field id="create-type">
+                <label>
+                  <span class="label">Type<span aria-hidden="true">*</span></span>
+                  <select id="expression-type-dropdown"
+                          ngDefaultControl 
+                          (change)="displayExpressionOutput()"
+                          formControlName="type">
+                    <option value='user'>User</option>
+                    <option value='team'>Team</option>
+                    <option value='token'>Token</option>
+                    <option value='*'>All members (*)</option>
+                  </select>
+                </label>
+                <chef-error *ngIf="expressionForm.get('type').hasError('required') && expressionForm.get('type').touched">
+                  Type is a required field.
+                </chef-error>
+              </chef-form-field>
+
+              <chef-form-field 
+                id="create-identity" 
+                *ngIf="showInputs('identity')"
+                [@dropInAnimation]="{ value: triggerValue, params: { height: 74.6875 }}" >
+                <label>
+                  <span class="label">Identity Provider<span aria-hidden="true">*</span></span>
+                  <select id="expression-identity-dropdown" 
+                          ngDefaultControl 
+                          (change)="displayExpressionOutput()"
+                          formControlName="identity">
+                    <option value="ldap">LDAP</option>
+                    <option value="local">local</option>
```

Comment:
The order LDAP, local, SAML seems odd. I would suggest local, LDAP, SAML.
@susanev Did you have a strong reason for this existing order?

Reply:
it's alphabetical (natural) which is what we've been trying to standardize on

Reply:
"Alphabetical with special circumstances", seems preferable. 🤷‍♂

Reply:
@susanev / @msorens - Is there a final consensus here?

Reply:
@SEAjamieD keep the order as defined in the issue for now, we can iterate later if we receive user feedback